pkg/controller/controller_instance_test.go

```diff
@@ -79,6 +81,8 @@ func TestReconcileInstanceNonExistentServiceClass(t *testing.T) {
 	}
 }
 
+// TestReconcileInstanceNonExistentBroker tests reconciling an instance whose
+// broker does not exist.  This returns an error.
```

Comment:
The conditions cause   testController.reconcileInstance(instance)   to return an error (i.e., its an expected result)